app/views/activity_mailer/activity_notification.text.erb

```diff
@@ -0,0 +1,6 @@
+<%= @activity %>
+
+<%= t("mailer.proposal_link_text",
+      proposal_url: proposal_url(@proposal)) %>
+
+<%= t("mailer.footer", feedback_url: feedback_url) %>
```

Comment:
oh hah I didn't even see this was text. I haven't been including the feedback link in text emails -- we should do that! added a card https://trello.com/c/Un6KSc7U